hoomd/md/force.py

```diff
@@ -271,73 +280,48 @@ class active(_Force):
 
```

Comment:
I think the note about the lack of MPI execution support should be removed. The new code will work in MPI.